eclass-writing/text.xml

```diff
@@ -644,10 +682,10 @@ domacosapp() {
 
 <p>
 An eclass may provide default implementations for any of the ebuild phase
-functions (<c>src_unpack</c>, <c>pkg_postinst</c> etc). This can be done either as a
+functions (<c>src_unpack</c>, <c>pkg_postinst</c>, etc). This can be done either as a
 simple function definition (which is not multiple eclass friendly) or using
 <c>EXPORT_FUNCTIONS</c>. Functions given to <c>EXPORT_FUNCTIONS</c> are implemented
-as normal, but have their name prefixed with <c>${ECLASS}_</c>.
+as normal, but have their name prefixed ("namespaced") with <c>${ECLASS}_</c>.
```

Comment:
"Namespaced"? Even if this is a word (it's neither in the Macmillan nor in the Merriam-Webster dictionary), it looks strange. It also doesn't add much information, the description is quite clear without it. My suggestion would be to drop this change.

Reply:
https://en.wiktionary.org/wiki/namespace#Verb
Please, let's not bikeshed this to hell. This is a *very* common term, it's in my company's style guide.

Reply:
This is common jargon which developers are going to use, let's keep it.